cves/kernel/CVE-2016-0823.yml

```diff
@@ -131,10 +134,12 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: |
+    This hardware problem was unknown until a year after this was
```

Comment:
Can you expand on exactly where the unit tests were here?

Reply:
i was unable to locate them